### ISSUE TITLE
Lack of accessibility under windows

### ISSUE THREAD

**Issue by DavidBitSource** [gpodder/gpodder#793]
It is not possible to use this program under Windows with a screen reader, since UIA or at least MSAA are not supported.

**Comment by auouymous** [comment]
Do other Gtk apps support those on Windows?

**Comment by DavidBitSource** [comment]
I did the last extensive research in 2011, back then they were not supporting MSAA.
Probably this indicates, that the issue is still current.
I intend to bring this to the developers of the GTK and would appreziate your support, since gPodder is an interesting project, where noone should be prevented of using it.

**Comment by elelay** [comment]
Yes, we are dependent on the gui toolkit for accessibility support.
Meanwhile, depending on the tasks you want to do, there is the command-line gpo-cmd.exe.
One can subscribe to feeds, manage extensions, download episodes, etc.
I'm interested in your feedback :-)
